### ISSUE TITLE
user integration with old drupal profile should import more user details

### ISSUE THREAD

**Issue by OriHoch** [Midburn/spark#32]
after #29 drupal users can log-in to spark

these drupal users have some data which is missing

most important:
* first_name
* last_name
* gender

possibly other data as well..

best way will be to add those fields to drupal_users table

**Comment by amotenko** [comment]
Nitay was adding some scripts, I'll ask him to update his progress here

**Comment by OriHoch** [comment]
I did not see a fix for this issue.. why is it closed?

**Comment by amotenko** [comment]
I'm sorry, I should have commented on that.
I see it as a duplicate of this broader issue: 

https://github.com/Midburn/Spark/issues/122

**Comment by OriHoch** [comment]
this issue is about data we are missing from drupal users (like we don't have their first_name / last_name fields for them)

even if we have SSO we will still need to figure out how to map the fields from drupal

for example, in spark we have 3 fields for name: name, first_name, last_name - but drupal only have "name" - how do we map and integrate those fields? should we split the name field by space? should we remove first_name / last_name from spark users table?

**Comment by amotenko** [comment]
Right, so since we decided to leach from the drupal database (for this year), we will effectively have no need for the user table.

Furthermore, we will need to cache every single user uid we fetch so that it can serve as a primary key in other tables.
Unfortunately we will not be able to cache any other information about the user since the drupal data is live and might change at any time.

Having said that, we will need to decide eventually if we go with name or first name / last name. I think separating them is a good thing because you can then address your users formally and informally. Next year when we migrate, we can try to split the name on a best effort basis and then when users login present them with a 'confirm your details' sort of a page :)
